GiveNGo/GiveNGo
# Update TSConfig & Package.json

## Description:
- These changes are updates made to the application's tsConfig file. Additionally subdirectories have been excluded when running the tsc command. Additional changes were made to the package.json file as additional dependencies were installed.  

## Changes I Made:
- Installed **Nodemon** dependency 
- Created NPM script to run application and compile to **Typescript**
- Created NPM script to start Nodemon in watch mode on our **server.js** file 
- Exclude proper subdirectories uncluding: Node_Modules, JestConfig, and Metro Bundler 

## How to Test:
- `npm install`
- `npm run both:tsc`
- The application should run and load the Metro Bundler

<img width="502" alt="Screen Shot 2020-07-20 at 8 33 01 PM" src="https://user-images.githubusercontent.com/55554784/88009803-37ffc380-cac8-11ea-8714-00d55e43623f.png">
